include/wx/html/htmprint.h

```diff
@@ -296,6 +307,8 @@ class WXDLLIMPEXP_HTML wxHtmlEasyPrinting : public wxObject
     wxString m_Headers[2], m_Footers[2];
     wxWindow *m_ParentWindow;
 
+    DialogMode m_dialog;
```

Comment:
Sorry for nitpicking, but wouldn't be better to name the variable `m_dialogMode` instead?